Makefile

```diff
@@ -30,13 +15,21 @@ LDFLAGS	=\
 	-lsqlite3\
 	-bind_at_load
 
-GCC_ARM = $(GCC_BIN) -Os -Wimplicit -isysroot $(SYSROOT) $(ARCH_FLAGS)
+GCC_ARM = $(GCC_BIN) -Os -Wimplicit -isysroot $(SDK) $(ARCH_FLAGS)
 
-default: main.o 
-	$(GCC_ARM) $(LDFLAGS) main.o -o keychain_dumper
+default: main.o list
+	@$(GCC_ARM) $(LDFLAGS) main.o -o keychain_dumper
 
 main.o: main.m
 	$(GCC_ARM) -c main.m
 
 clean:
 	rm -f keychain_dumper *.o 
+
+list:
+	security find-identity -pcodesigning -v
```

Comment:
I'm not super familiar with `find-identity`, but it looks like `-v` will only return "valid" code signing certificates. That seems to exclude self-signed certificates (as are used in the rest of the Readme). I'm assuming that self-signed still works fine (give this is being run on a jail broken device), right? If so, can you update this to remove the` -v`?